src/lib/FormGroupLayout.tsx

```diff
@@ -17,16 +18,24 @@ const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>)
     formState: { errors },
   } = useFormContext();
 
+  const { requiredFields } = useInternalFormContext();
+
   const fieldError = get(errors, name) as FieldError | undefined;
   const errorMessage = String(fieldError?.message);
 
+  let labelText = String(label);
```

Comment:
This breaks existing functionality. 

The label is of type `ReactNode`. 

So it can be a string but it doesn't have to. 

I would only add the asteriks if the label is of type string. 

Somebody might pass `<strong>MyLabel</strong>` as label. Than we have no chance to add the asteriks.

Reply:
@neotrow Done :D